Auto update failing for libpoco

Hi, I'm Termux 🤖.

I'm here to help you update your Termux packages.

I've tried to update the [libpoco](https://github.com/termux/termux-packages/blob/6ae55694d955751b489449f0b3cec8d71ac73cfc/packages/libpoco) package, but it failed.

Here's the output of the update script:
<details>
<summary>Show log</summary>
<pre lang="console">
INFO: Updating libpoco [Current version: 1.13.3]
INFO: package being updated to 1.14.0.
Downloading https://github.com/pocoproject/poco/archive/refs/tags/poco-1.14.0-release.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

100 5024k    0 5024k    0     0  5070k      0 --:--:-- --:--:-- --:--:-- 5070k
100 11.5M    0 11.5M    0     0  6516k      0 --:--:--  0:00:01 --:--:-- 8242k

You are about to commit these changes:

--------------------
diff --git a/packages/libpoco/build.sh b/packages/libpoco/build.sh
index c307787c57..650a8ff2d3 100644
--- a/packages/libpoco/build.sh
+++ b/packages/libpoco/build.sh
@@ -2,10 +2,9 @@ TERMUX_PKG_HOMEPAGE=https://pocoproject.org/
 TERMUX_PKG_DESCRIPTION="A comprehensive set of C++ libraries that cover all modern-day programming needs"
 TERMUX_PKG_LICENSE="BSL-1.0"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION="1.13.3"
-TERMUX_PKG_REVISION=1
+TERMUX_PKG_VERSION="1.14.0"
 TERMUX_PKG_SRCURL=https://github.com/pocoproject/poco/archive/refs/tags/poco-${TERMUX_PKG_VERSION}-release.tar.gz
-TERMUX_PKG_SHA256=9f074d230daf30f550c5bde5528037bdab6aa83b2a06c81a25e89dd3bcb7e419
+TERMUX_PKG_SHA256=48e98bbc8ed9fcd8bec43538402181d33cc2abe7e19ac8aea4bc113c9ba8ef89
 TERMUX_PKG_AUTO_UPDATE=true
 TERMUX_PKG_UPDATE_VERSION_REGEXP='(?<=-)[^-]+(?=-)'
 TERMUX_PKG_DEPENDS="libc++, libexpat, libsqlite, openssl, pcre2, zlib"
--------------------

bump(main/libpoco): 1.14.0


Not committing to Git!

INFO: Trying to build package.
INFO: Package libpoco exists in main repo.
Running container 'termux-package-builder' from image 'ghcr.io/termux/package-builder'...
termux - building libpoco for arch aarch64...
Downloading https://packages-cf.termux.dev/apt/termux-main/dists/stable/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13133  100 13133    0     0  39408      0 --:--:-- --:--:-- --:--:-- 39438
Downloading https://packages-cf.termux.dev/apt/termux-main/dists/stable/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0   2457      0 --:--:-- --:--:-- --:--:--  2464
gpg: Signature made Sun 01 Dec 2024 06:21:18 PM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
Downloading https://packages-cf.termux.dev/apt/termux-root/dists/root/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13320  100 13320    0     0  36289      0 --:--:-- --:--:-- --:--:-- 36294
100 13320  100 13320    0     0  36279      0 --:--:-- --:--:-- --:--:-- 36195
Downloading https://packages-cf.termux.dev/apt/termux-root/dists/root/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0   2364      0 --:--:-- --:--:-- --:--:--  2366
gpg: Signature made Thu 28 Nov 2024 12:30:35 PM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
Downloading https://packages-cf.termux.dev/apt/termux-x11/dists/x11/Release
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 13119  100 13119    0     0  35307      0 --:--:-- --:--:-- --:--:-- 35361
Downloading https://packages-cf.termux.dev/apt/termux-x11/dists/x11/Release.gpg
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   833  100   833    0     0   2227      0 --:--:-- --:--:-- --:--:--  2227
gpg: Signature made Sun 01 Dec 2024 06:32:24 PM UTC
gpg:                using RSA key CC72CF8BA7DBFA0182877D045A897D96E57CF20C
gpg: Good signature from "Termux Releases (Termux automatic builds) <contact@termux.dev>" [ultimate]
gpg:                 aka "Termux Releases (Termux automatic builds) <contact@termux.org>" [ultimate]
Downloading dependency libc++@27c if necessary...
Found libc++ in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libc++ to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libexpat@2.6.4 if necessary...
Found libexpat in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libexpat to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libandroid-support@29 if necessary...
Found libandroid-support in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libandroid-support to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ncurses@6.5.20240831-2 if necessary...
Found ncurses in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting ncurses to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ncurses-utils@6.5.20240831-2 if necessary...
Found ncurses-utils in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting ncurses-utils to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ncurses-ui-libs@6.5.20240831-2 if necessary...
Found ncurses-ui-libs in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting ncurses-ui-libs to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ncurses-ui-libs-static@6.5.20240831-2 if necessary...
Found ncurses-ui-libs-static in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting ncurses-ui-libs-static to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency readline@8.2.13 if necessary...
Found readline in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting readline to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency zlib@1.3.1 if necessary...
Found zlib in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting zlib to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency tcl@8.6.14 if necessary...
Found tcl in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting tcl to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libsqlite@3.47.1 if necessary...
Found libsqlite in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libsqlite to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency sqlite@3.47.1 if necessary...
Found sqlite in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting sqlite to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency libsqlite-tcl@3.47.1 if necessary...
Found libsqlite-tcl in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting libsqlite-tcl to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency ca-certificates@1:2024.09.24 if necessary...
Found ca-certificates in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting ca-certificates to /home/builder/.termux-build/_cache-all...
Downloading dependency ca-certificates-java@1:2024.09.24 if necessary...
Found ca-certificates-java in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting ca-certificates-java to /home/builder/.termux-build/_cache-all...
Downloading dependency openssl@1:3.3.2 if necessary...
Found openssl in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting openssl to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency openssl-tool@1:3.3.2 if necessary...
Found openssl-tool in https://packages-cf.termux.dev/apt/termux-main/dists/stable
extracting openssl-tool to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency pcre2@10.44 if necessary...
Found pcre2 in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/p/pcre2/pcre2_10.44_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  868k  100  868k    0     0  10.5M      0 --:--:-- --:--:-- --:--:-- 10.5M
extracting pcre2 to /home/builder/.termux-build/_cache-aarch64...
Downloading dependency pcre2grep@10.44 if necessary...
Found pcre2grep in https://packages-cf.termux.dev/apt/termux-main/dists/stable
Downloading https://packages-cf.termux.dev/apt/termux-main/pool/main/p/pcre2grep/pcre2grep_10.44_aarch64.deb
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100 33456  100 33456    0     0   752k      0 --:--:-- --:--:-- --:--:--  759k
extracting pcre2grep to /home/builder/.termux-build/_cache-aarch64...
Downloading https://github.com/pocoproject/poco/archive/refs/tags/poco-1.14.0-release.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

100 4864k    0 4864k    0     0  5605k      0 --:--:-- --:--:-- --:--:-- 5605k
100 11.5M    0 11.5M    0     0  6879k      0 --:--:--  0:00:01 --:--:-- 8169k
Applying patch: Foundation-src-NamedEvent_UNIX.cpp.patch
Applying patch: Foundation-src-NamedMutex_UNIX.cpp.patch
Applying patch: Foundation-src-Path_UNIX.cpp.patch
Applying patch: Foundation-src-SharedMemory_POSIX.cpp.patch
Downloading https://github.com/Kitware/CMake/releases/download/v3.31.1/cmake-3.31.1-linux-x86_64.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

100 52.4M  100 52.4M    0     0   128M      0 --:--:-- --:--:-- --:--:--  128M
Downloading https://github.com/ninja-build/ninja/releases/download/v1.12.1/ninja-linux.zip
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

100  120k  100  120k    0     0   869k      0 --:--:-- --:--:-- --:--:--  869k
Archive:  /home/builder/.termux-build/libpoco/tmp/ninja-1.12.1.zip
  inflating: /home/builder/.termux-build/_cache/ninja-1.12.1/ninja  
-- Android: Targeting API '24' with architecture 'arm64', ABI 'arm64-v8a', and processor 'aarch64'
-- The C compiler identification is Clang 18.0.3
-- The CXX compiler identification is Clang 18.0.3
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /home/builder/.termux-build/_cache/android-r27c-api-24-v1/bin/clang - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /home/builder/.termux-build/_cache/android-r27c-api-24-v1/bin/clang++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Checking for C++17 compiler
-- Checking for C++17 compiler - available
-- Found OpenSSL: /data/data/com.termux/files/usr/lib/libcrypto.so (found version "3.3.2")
-- Could NOT find APR (missing: APR_INCLUDE_DIR APR_LIBRARY) 
-- Could NOT find APRUTIL (missing: APRUTIL_INCLUDE_DIR APRUTIL_LIBRARY) 
-- Could NOT find Apache2 (missing: APACHE2_INCLUDE_DIR) 
-- Could NOT find MySQL (missing: MYSQL_INCLUDE_DIR MYSQL_LIBRARY) 
-- Could NOT find PostgreSQL (missing: PostgreSQL_LIBRARY PostgreSQL_INCLUDE_DIR) 
-- Could NOT find ODBC (missing: ODBC_LIBRARY ODBC_INCLUDE_DIR) 
-- Building without tests & samples
-- Using external sqlite, zlib, pcre2, expat, libpng, ...
-- Found PCRE2: /data/data/com.termux/files/usr/lib/libpcre2-8.so
-- Found ZLIB: /data/data/com.termux/files/usr/lib/libz.so (found version "1.3.1")
CMake Error at /home/builder/.termux-build/_cache/cmake-3.31.1/share/cmake-3.31/Modules/FindPackageHandleStandardArgs.cmake:233 (message):
  Could NOT find Utf8Proc (missing: UTF8PROC_LIBRARY UTF8PROC_INCLUDE_DIR)
Call Stack (most recent call first):
  /home/builder/.termux-build/_cache/cmake-3.31.1/share/cmake-3.31/Modules/FindPackageHandleStandardArgs.cmake:603 (_FPHSA_FAILURE_MESSAGE)
  cmake/FindUtf8Proc.cmake:49 (FIND_PACKAGE_HANDLE_STANDARD_ARGS)
  Foundation/CMakeLists.txt:32 (find_package)


-- Configuring incomplete, errors occurred!
ERROR: failed to build.</pre>
</details>

<hr>
<i>
Above error occured when I last tried to update at 2024-12-02 00:35:13 UTC.<br>
Run ID: <a href="https://github.com/termux/termux-packages/actions/runs/12110418545">12110418545</a><br><br>
<b>Note:</b> Automatic updates will be disabled until this issue is resolved.<br>
</i>

--- a/packages/libpoco/build.sh
+++ b/packages/libpoco/build.sh
@@ -2,12 +2,11 @@ TERMUX_PKG_HOMEPAGE=https://pocoproject.org/
 TERMUX_PKG_DESCRIPTION="A comprehensive set of C++ libraries that cover all modern-day programming needs"
 TERMUX_PKG_LICENSE="BSL-1.0"
 TERMUX_PKG_MAINTAINER="@termux"
-TERMUX_PKG_VERSION="1.13.3"
-TERMUX_PKG_REVISION=1
+TERMUX_PKG_VERSION="1.14.0"
 TERMUX_PKG_SRCURL=https://github.com/pocoproject/poco/archive/refs/tags/poco-${TERMUX_PKG_VERSION}-release.tar.gz
-TERMUX_PKG_SHA256=9f074d230daf30f550c5bde5528037bdab6aa83b2a06c81a25e89dd3bcb7e419
+TERMUX_PKG_SHA256=48e98bbc8ed9fcd8bec43538402181d33cc2abe7e19ac8aea4bc113c9ba8ef89
 TERMUX_PKG_AUTO_UPDATE=true
 TERMUX_PKG_UPDATE_VERSION_REGEXP='(?<=-)[^-]+(?=-)'
-TERMUX_PKG_DEPENDS="libc++, libexpat, libsqlite, openssl, pcre2, zlib"
+TERMUX_PKG_DEPENDS="libc++, libexpat, libsqlite, openssl, pcre2, utf8proc, zlib"
 TERMUX_PKG_FORCE_CMAKE=true
 TERMUX_PKG_EXTRA_CONFIGURE_ARGS="-DPOCO_UNBUNDLED=ON"

